src/combinatorics/quiver/single_obj.lean

```diff
@@ -0,0 +1,111 @@
+/-
+Copyright (c) 2022 Antoine Labelle. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Labelle
+-/
+import combinatorics.quiver.connected_component
+
+/-!
+# Single-object quiver
+
+Single object quiver with a given arrows type.
+
+## Main definitions
+
+Given a type `α`, `single_obj α` is `unit` type, whose single object is called `star α`, with
```

Comment:
```suggestion
Given a type `α`, `single_obj α` is the `unit` type, whose single object is called `star α`, with
```

Reply:
I think this is worth fixing. This sentence does not read properly.